roosevelt.js

```diff
@@ -182,6 +182,8 @@ module.exports = function (params) {
       require('./lib/htmlValidator')(app, mapRoutes)
     }
 
+    buildScanner(app)
```

Comment:
this will not execute in the order you expect

look at the code preceding this. each function calls the next

you'll want to follow the pattern. define a function and call it from `validateHTML`, then have your function call `mapRoutes` the way `validateHTML` was